templates/scenarios/ts/notification-http-timer-trigger/teamsapp.yml.tpl

```diff
@@ -27,9 +27,11 @@ provision:
 # Triggered when 'teamsfx deploy' is executed
 deploy:
   - uses: cli/runNpmCommand
+    name: install dependencies
```

Comment:
Is there any sample how the `name` will be shown in output?
Will the `name`s be added to all other actions as well?
And, it seems only added to TS templates but not all JS templates.

Reply:
Tt may shows like this in log output:
```
Lifecycle stage: deploy(8 step(s) in total). The following actions will be executed:
(1/8) Action cli/runNpmCommand(install dependencies): running npm command.
(2/8) Action cli/runNpmCommand(build app): running npm command.
....
```

We add `name` in actions only when two or more same drivers appeared in one lifecycle.
So I added in TS templates not JS templates.